css/common.css

```diff
@@ -1,64 +1,60 @@
 @font-face {
-        font-family: "Spirax";
-        src: url("../css/Spirax-Regular.ttf")
+    font-family: "Spirax";
+    src: url("../css/Spirax-Regular.ttf")
 }
 
-body, ul, li, div, span, p, strong {
-  margin:0;
-  padding:0;
-  position:relative;
-}
 body {
-  background: url(../img/body.png) repeat-y 0 0;
-  font-family: 'Spirax', Helvetica, Arial;
-  display:block;
-  min-width:960px;
-  position:relative;
+    margin:0;
+    padding:0;
+    font-family: 'Spirax', Helvetica, Arial;
+    display: flex;
 }
-#side,#main {
-  top:0;
+
+@media all and (max-width: 700px) {
+  body {
+      display: block;
+  }
 }
```

Comment:
This is moving from flexbox layout, to block layout on small viewport (under 700px). This is layout dependent. In the case of the website under `700px` it becomes uncomfortable to read the text on the right part. One problem that arise also, is that the logo is cropped. I tried to go the minimal changes.